inc/Composer/Command.php

```diff
@@ -171,8 +184,20 @@ protected function logs( InputInterface $input, OutputInterface $output ) {
 			'VOLUME' => getcwd(),
 			'COMPOSE_PROJECT_NAME' => basename( getcwd() ),
 		] );
+		$compose->setTimeout( 0 );
```

Comment:
curious what this is for?

Reply:
When tailing logs, the command is kept running, so we don't want to abort after 60 seconds